__tests__/app.test.js

```diff
@@ -412,4 +412,15 @@ describe("/api/comments/:comment_id", () => {
   });
 });
 
+describe("/api", () => {
+  test("GET:200 sends the API info object back to the client", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.APIs).toBeInstanceOf(Object);
+      });
+  });
+});
+
 afterAll(() => db.end());
```

Comment:
we would normally see this at the top of the file after the before each, it makes no difference, just strange to see down at the bottom of the file 😄

Reply:
will move it and leave it at the top from now on